apps/pm-user/src/pages/login/Login.js

```diff
@@ -20,39 +21,26 @@ const Login = () => {
             sessionToken,
           });
         }
-      });
-
-  const logoutHandler = () => oktaAuth.signOut('/');
+      })
+      .catch((err) => console.log('vinit', err));
 
   if (!authState) {
     return <div>Loading...</div>;
   }
-
   if (!authState.isAuthenticated) {
     return (
-      <div className={styles.superContainer}>
-        <div className={styles.leftContainer}></div>
-        <div className={styles.rightContainer}>
-          <div className={styles.loginBox}>
-            <div className={styles.logInLogo}>Pesto Matrimony</div>
-            <div className={styles.signInText}>Sign in</div>
-            <LoginForm onLogin={loginHandler} />
-            <p className={styles.paragraph}>Forgot Password</p>
-            <p className={styles.paragraph}>
-              Don't have an account? <span className={styles.link}>SignUp</span>
-            </p>
-          </div>
-        </div>
-      </div>
+      <FormWrapper
+        bottomText="Dont have account?"
```

Comment:
"Don't have an account ?" would be correct here.

Reply:
corrected